packages/components/src/utils/meta/meta.test.tsx

```diff
@@ -0,0 +1,79 @@
+import * as fs from "fs/promises";
+import * as allExports from "../../index";
+
+describe("meta", () => {
+  // If this test fails, please update meta.json accordingly.
+  it("verifies that the meta.json file is up to date", async () => {
```

Comment:
nice. I was going to ask about how we might ensure the meta entires stay up to date.

I had been thinking about it the other way around, where we have a script like `findComponentNamesDeep` and use that to write the `meta.json` file as part of the build step or something which might require less manual intervention

Reply:
fair enough. adding an entry for your new component when you get a failing test seems like a trivial task. I'm happy with this solution!